### ISSUE TITLE
Copy of release docs builds are not being pushed to latest/ directory in S3 bucket by travis during release builds

### ISSUE THREAD

**Issue by davidlmobley** [choderalab/yank#810]
The link to quickstart instructions on the main page (main README.md) is broken.

**Comment by jchodera** [comment]
@Lnaden : For some reason, the travis release builds are not pushing the most up-to-date release docs to http://getyank.org/latest. This used to work. Can you fix this?

Updating the issue name to reflect the true issue.

**Comment by Lnaden** [comment]
@jchodera This is something we discussed in person for choderalab/yank#601 was that the only way to have a "latest" and "development" and a "most recent version" would result in a large amount of file duplication. Since we pay by how much we upload, we decided erred on the side of less duplication to keep file sizes down. So we opted to have just "latest" redirect to the most recent stable version head, but I could never figure out how to do a HTTP_REDIRECT on S3, which near as I can tell is impossible; I have been looking into this on and off since I made that change. So the way its setup now is "getyank.org/latest" routes to the most recent stable version, then all navigation would have to be done after that.

I thought I got all the dead links, but apparently I missed that one, which is also why I have a card for [site review on the project board](https://github.com/choderalab/yank/projects/1#card-4355732)

I can make the change to have duplication if its desired, or we could think of another way to host the site rather than S3.
